src/library.rs

```diff
@@ -848,16 +848,16 @@ impl Type {
         }
     }
 
-    pub fn is_fundamental(&self) -> bool {
+    pub fn is_basic(&self) -> bool {
         matches!(*self, Type::Fundamental(_))
```

Comment:
I wonder if I should rename the enum variant from Type::Fundamental to Type::Basic

Reply:
That would make it coherent with the function name at least.

Reply:
Done